build.gradle.kts

```diff
@@ -86,6 +90,33 @@ yamlValidator {
     isSearchRecursive = true
 }
 
+sonarqube {
+    // If you want to use this logcally a sonarLogin has to be provide, either via Username and Password
+    // or via token, https://docs.sonarqube.org/latest/analysis/analysis-parameters/
+    properties {
+        // Default properties if somebody wants to execute it locally
+        property("sonar.projectKey", "junit-pioneer_junit-pioneer")
+        property("sonar.organization", "junit-pioneer-xp")
+        property("sonar.host.url", "https://sonarcloud.io")
+
+        // reminder to @aepfli, provide a follow up pull request do remove this line, and delete token in sonarcloud
+        property("sonar.login", "a175fc2e92f951bbc1da3de77f3309d85dfa5ddd")
```

Comment:
this is my personal token, and it is just for uploading managing sonar, it seems like that travis ci is not populating the sonar properties via the add on, i assume that this needs to be in master first, to be correctly populated.